submissions/kissik/data-structures/index.js

```diff
@@ -0,0 +1,127 @@
+const http = require('http');
+
+const fs = require('fs');
+
+const port = process.env.PORT || 3210;
+
+const page404 = {
+  path: 'pages/404.html',
+  type: { 'Content-Type': 'text/html' },
+  verb: 404
+};
+
+const action = {
+  type: { 'Content-Type': 'text/html' },
+  verb: 200
+};
+
+const contentType = {
+  '/': {
+    path: 'pages/index.html',
+    type: { 'Content-Type': 'text/html' },
+    verb: 200
+  },
+  '/css/theme.css': {
+    path: 'css/theme.css',
+    type: { 'Content-Type': 'text/css' },
+    verb: 200
+  },
+  '/img/binoculars.jpg': {
+    path: 'img/binoculars.jpg',
+    type: { 'Content-Type': 'image/jpeg' },
+    verb: 200
+  }
+};
+
+const makeHTML = data => {
+  return `<html><head>
+                    <link rel="stylesheet" type="text/css" href="css/theme.css">
+                    </head>
+                    <body><h1>Your data structure:</h1>
+                    <p class="answer">${data}</p>
+                    <a href='/'>return</a>
+            </body></html>`;
+};
+
+const list = {};
+
+list.data = {};
+
+list.push = data => {
+  list.data = {
+    item: data,
+    next: list.data
+  };
+};
+
+list.pop = () => {
+  const itemValue = list.data.item ? list.data.item : 'none';
+  if (list.data) list.data = list.data.next;
+  return itemValue;
```

Comment:
I don't think that returning of string `'none'` is a good idea. 

Basically, list - is just a structure and can contain any type of data. If you will return `string` type, we can assume that we have strings in it.

Also, this will requires from developer to know contract of this concrete list and all strings values which can be returned from it - `'none'`, `'another string'`, `'why do we need strings?'` :) 

Instead, we can use existing approach - `undefined` can do a trick because it means `empty` in javascript and commonly used here and there. As it was done in `Array#pop`, as an example.